### PR TITLE
Adding liquibase config for oauth_resource_id_auths table 

### PR DESCRIPTION
**Pull request by isuftin**
Also switching base container to use DockerHub

### CODE REVIEW

**Comment by isuftin** [comment]
@zmoore-usgs Take a look at the liquibase updates here

**Comment by isuftin** [comment]
Also note we are now using the DockerHub source for the spring-boot base image

File: liquibase/liquibase.sh [hunk 1]
```diff
@@ -1,3 +1,6 @@
 #!/bin/bash
 
-docker run --rm --network container:database -v "$(pwd)":/liquibase/ --env-file liquibase.env webdevops/liquibase:mysql update
+# Figure out the directory this script sits in
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
```
Comment (zmoore-usgs) [inline liquibase/liquibase.sh]: Just out of curiosity - What's the benefit of this method versus what we had before?

Comment (isuftin) [reply liquibase/liquibase.sh]: @zmoore-usgs When running this bash script now, you can do it from any directory. Previously you had to be in the same directory as the bash script. 

This is a good piece of code to copy/paste into any bash script where you don't want to be forced to be in the same dir as the script when executing it.

**Pull request merged**